README.md

```diff
@@ -1 +1,146 @@
-# REANA example - CMS ttbar analysis with Coffea
+# REANA example - AGC CMS ttbar analysis with Coffea
+
+This [REANA](http://www.reana.io/)_ reproducible analysis example demonstrates
+a  [AGC](https://arxiv.org/abs/1010.2506) - Analysis Grand Challenge.
+
+## Analysis Grand Challenge
+
+For full explanation please have a look at this documentation:
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7274936.svg)](https://doi.org/10.5281/zenodo.7274936)
+[![Documentation Status](https://readthedocs.org/projects/agc/badge/?version=latest)](https://agc.readthedocs.io/en/latest/?badge=latest)
+
+The Analysis Grand Challenge (AGC) is about performing the last steps in an analysis pipeline at scale to test workflows envisioned for the HL-LHC.
+This includes
+
+- columnar data extraction from large datasets,
+- processing of that data (event filtering, construction of observables, evaluation of systematic uncertainties) into histograms,
+- statistical model construction and statistical inference,
+- relevant visualizations for these steps,
+
+The physics analysis task is a $t\bar{t}$ cross-section measurement with 2015 CMS Open Data (see `datasets/cms-open-data-2015`).
+The current reference implementation can be found in `analyses/cms-open-data-ttbar`.
+
+### 1. Input data for ``reana.yaml`` 
+This file describes the above analysisc structure with its inputs, code, runtime environment, computational workflow steps and expected outputs.
+
+The analysis takes the following inputs:
+
+- ``nanoAODschema.json`` input `.root` files
+- ``Snakefile`` The Snakefile for 
+- ``ttbar_analysis_reana.ipynb`` The main notebook file where files are processed and analysed.
+- ``file_merging.ipynb`` Notebook to merge each processed `.root` file in one file with unique keys.
+- ``final_merging.ipynb`` Notebook to merge histograms together all of 
+
+### 2. Compute environment 
+
+To be able to rerun the AGC after some time, we need to
+"encapsulate the current compute environment", for example to freeze the ROOT version our
+analysis is using. We shall achieve this by preparing a [Docker](https://www.docker.com/)
+container image for our analysis steps.
+
+We are using the modified verison of the ``analysis-systems-base`` [Docker image](https://github.com/iris-hep/analysis-systems-base) container with additional packages, the main on is [papermill](https://papermill.readthedocs.io/en/latest/) which allows to run the Jupyter Notebook from the command line with additional parameters.
+
+In our case, the Dockerfile creates a conda virtual environment with all necessary packages for running the AGC analysis.
+
+```console
+$ less environment/Dockerfile
+```
+
+We can build our AGC environment image and give it a name
+`docker.io/reanahub/reana-demo-agc-cms-ttbar-coffea`:
+
+```console
+$ docker build -f environment/Dockerfile -t docker.io/reanahub/reana-demo-agc-cms-ttbar-coffea .
```

Comment:
Please amend the command to be in the `environment` directory. Also, you can build version `1.0.0` of the image instead of `latest`, so please use trailing `:1.0.0`.